core/src/main/java/org/sonarsource/sonarlint/core/analyzer/sensor/LtsApiSensorContext.java

```diff
@@ -105,8 +106,8 @@ public boolean matchFilePattern(String antPattern) {
   private final Project project;
 
   public LtsApiSensorContext(Project project, InputModule inputModule, Settings settings, FileSystem fs, ActiveRules activeRules, SensorStorage sensorStorage,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)